runtime/src/prioritization_fee_cache.rs

```diff
@@ -410,7 +410,32 @@ impl PrioritizationFeeCache {
                     .unwrap_or_default();
                 for account_key in account_keys {
                     if let Some(account_fee) =
-                        slot_prioritization_fee.get_writable_account_fee(account_key)
+                        slot_prioritization_fee.get_min_writable_account_fee(account_key)
+                    {
+                        fee = std::cmp::max(fee, account_fee);
+                    }
+                }
+                (*slot, fee)
+            })
+            .collect()
+    }
+
+    pub fn get_prioritization_fees2(
```

Comment:
What's the benefit of all the duplicate methods? If we've written the apis right, I would think a `get_transaction_fee(0)` should yield the same result as `get_min_transaction_fee()`, so we can just update `get_prioritization_fees` to take `percentile`